pkg/client/localsubjectaccessreview.go

```diff
@@ -63,11 +64,12 @@ func (c *localSubjectAccessReviews) Create(sar *authorizationapi.LocalSubjectAcc
 		}
 		deprecatedResponse := &authorizationapi.SubjectAccessReviewResponse{}
 
-		req, err := overrideAuth(c.token, c.r.Post().Namespace(c.ns).Resource("subjectAccessReviews"))
+		var deprecatedReq *restclient.Request
```

Comment:
yes you are correct. I have re-done it with a different name for the second error, it is clearer.